influxql/iterator.gen.go

```diff
@@ -1287,6 +1287,11 @@ func (itr *floatExprIterator) Next() (*FloatPoint, error) {
 			if itr.points == nil {
 				continue
 			}
+
+			if b == nil || b.Nil {
```

Comment:
Can you modify this so it's `} else if b == nil {`?

Also, if b is not nil so `b.Nil` is true, that's fine. We don't want to exclude that since that's a valid value. We only want to restrict when `b == nil`.